.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
```

Comment:
The recent version is v6.0 , I guess the bot will update it once we merge this PR?